.github/workflows/quality-check.yaml

```diff
@@ -21,11 +21,34 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-          python-version: [3.6, 3.7, 3.8]
-          tensorflow: ["~=2.5.0"]
+          python-version: ["3.7", "3.8", "3.9", "3.10"]
+          tensorflow: ["==2.5", "==2.6", "==2.7", "==2.8"]
+          include:
+            - tensorflow: "==2.5"
+              keras: "==2.6"
```

Comment:
Why do we have to explicitly specify and install Keras? Is it because as of [TF2.6](https://github.com/tensorflow/tensorflow/releases/tag/v2.6.0), Keras moved to its own repo again?

Reply:
> Why do we have to explicitly specify and install Keras? Is it because as of [TF2.6](https://github.com/tensorflow/tensorflow/releases/tag/v2.6.0), Keras moved to its own repo again?

Yes, and TensorFlow don't always install the right version, so we have to help it.